src/django/api/models/state.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.contrib.gis.db import models as gis_models
+
+__all__ = ["State"]
+
+
+class State(models.Model):
+    id = models.CharField(max_length=2, primary_key=True, unique=True)
+    name = models.CharField(max_length=127)
+    boundary = gis_models.PolygonField(geography=True)
+    options = models.JSONField(blank=True, null=True)
```

Comment:
Instead of `django.db.models.JSONField`, prefer `django.contrib.postgres.fields.JSONField`, to take advantage of Postgres-native JSON support.

Reply:
Also consider adding `help_text` explaining that this JSON dictionary will contain state-specific configuration.

Reply:
Documenting here since this was a separate slack discussion, the postgres `JSONField` is deprecated and will lose support in Django 4.0 so we'll move forward with the recommended `django.db.models.JSONField`. I'll still update to add `help_text` to this field, though.